docs/src/intro.md

```diff
@@ -125,3 +126,21 @@ docker-compose up
 ```
 
 *Quality-time* is served at [http://localhost](http://localhost). Use username `jadoe` and password `secret` to log in.
+
+# Why *Quality-time*?
+
+ICTU developed *Quality-time* to help projects and teams within ICTU gain actionable insight into the quality of the software they are developing and maintaining. The current vision on the scope of *Quality-time* is:
+
+- *Quality-time* collects data from sources that projects and teams are using anyway to develop and maintain software. *Quality-time* does not analyse the software itself.
```

Comment:
the word "anyway" seems redundant, how about:
`that projects and teams use in developing and maintaining software`

Reply:
It's not redundant IMO. Compare to the Dutch: "Quality-time verzamelt gegevens uit bronnen die projecten en team sowieso gebruiken om software te ontwikkelen en onderhouden". Anyway/sowieso stress that the (data in the) tools are reused for the purpose of quality management.